format/spec.md

```diff
@@ -1480,6 +1497,9 @@ This serialization scheme is for storing single values as individual binary valu
 | **`struct`**                 | Not supported                                                                                                |
 | **`list`**                   | Not supported                                                                                                |
 | **`map`**                    | Not supported                                                                                                |
+| **`geometry`**               | A single point, encoded as a {x, y, optional z, optional m} concatenation of its 8-byte little-endian IEEE 754 coordinate values, with the optional coordinates encoded as NaN if unset. |
+| **`geography`**              | A single point, encoded as a {x, y, optional z, optional m} concatenation of its 8-byte little-endian IEEE 754 coordinate values, with the optional coordinates encoded as NaN if unset. |
```

Comment:
This section is for `lower_bound` and `upper_bound`

> This serialization scheme is for storing single values as individual binary values in the lower and upper bounds maps of manifest files.

Reply:
Yea this was @rdblue suggestion, because the lower_bound and upper_bound is a point, then we can make it more smaller than WKB, which is signfiicant because this is on every manifest entry.  The comment is here https://github.com/apache/iceberg/pull/10981#discussion_r1768947092 , we had another discussion but cant find it.